SonarQube.Client/Api/V7_20/GetIssuesRequest.cs

```diff
@@ -102,8 +105,15 @@ protected override SonarQubeIssue[] ParseResponse(string response)
         private IssueLocation ToIssueLocation(ServerIssueLocation serverIssueLocation) =>
             new IssueLocation(ComputePath(serverIssueLocation.Component), serverIssueLocation.Component, ToIssueTextRange(serverIssueLocation.TextRange), serverIssueLocation.Message);
 
-        private static IssueTextRange ToIssueTextRange(ServerIssueTextRange serverIssueTextRange) =>
-            new IssueTextRange(serverIssueTextRange.StartLine, serverIssueTextRange.EndLine, serverIssueTextRange.StartOffset, serverIssueTextRange.EndOffset);
+        private static IssueTextRange ToIssueTextRange(ServerIssueTextRange serverIssueTextRange)
+        {
+            return serverIssueTextRange == null
```

Comment:
I checked and it seems that textRange field is available in 7.2+.
It indeed returns null when the issue is at file-level, same as the way Line was null.
E.g.:
https://peach.sonarsource.com/project/issues?id=akka-msbuild14&resolved=false&rules=csharpsquid%3AS4026